Database/Corrections/sodQuestFixes.lua

```diff
@@ -3396,6 +3396,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425344,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90222] = {
+            [questKeys.name] = "Dreamstate",
+            [questKeys.startedBy] = {{12977,4636,4632,4637,4634,4633,12976,4635},{450014,450015}},
```

Comment:
I think this is another case of "do not show all NPCs", to not clutter the map more than necessary.